public/js/actions/subscriptions.js

```diff
@@ -33,43 +33,68 @@ export function getUserSubscriptions(jquery=$) {
 }
 
 
-export function createSubscription(braintreeToken, productId,
-                                   creditCard, jquery=$,
+export function createSubscription(productId, payMethod,
+                                   braintreeToken, jquery=$,
                                    BraintreeClient=braintree.api.Client) {
```

Comment:
or to keep the other required args in place, it could be:
````
createSubscription(productId, braintreeToken, 
                   {payMethod=undefined, creditCard=undefined, 
                    jquery=$, BraintreeClient=braintree.api.Client}) {
}
````